static/src/javascripts/projects/common/modules/ui/first-pv-consent-banner.js

```diff
@@ -135,4 +143,13 @@ export const _ = {
     bindableClassNames,
 };
 
-export { firstPvConsentBanner };
+export {
+    firstPvConsentBanner,
+    canShow,
+    track,
+    bindClickHandlers,
+    messageCode,
+    makeHtml,
+    template,
+    bindableClassNames,
+};
```

Comment:
this is quite a long list of exports, could we try to self-contain this inside this component a bit more?

Reply:
I've changed your `makeHtml` so the template and bindableClassNames are not passed in as parameters but simply referenced in the closure scope. This makes sense to me since I find it hard to imagine how this function could be re-used with these values changing. This reduces the export list by 2 functions